featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -346,10 +347,19 @@ async def handle_online_disabled_features(
             )
 
             if updated_feature_ids:
-                updated_feature_table = await self._update_offline_store_feature_table(
-                    feature_table_dict,
-                    updated_feature_ids,
-                )
+                try:
+                    updated_feature_table = await self._update_offline_store_feature_table(
+                        feature_table_dict,
+                        updated_feature_ids,
+                    )
+                except OfflineStoreFeatureTableBadStateError:
+                    logger.error(
+                        "Failed to update offline store feature table when disabling a deployment",
+                        exc_info=True,
+                        extra={"table_name": feature_table_dict["name"]},
+                    )
+                    await self._delete_offline_store_feature_table(feature_table_dict["_id"])
+                    continue
```

Comment:
Should we also delete the precomputed lookup feature table?

Reply:
Good catch, we should delete those as well.

Reply:
Actually `_delete_offline_store_feature_table` already handles deleting lookup feature tables associated with the feature table.